### DISCUSSION TITLE
Shiny.Extensions.Push Firebase legacy FCM APIs deprecation

### DISCUSSION THREAD

**Discussion by Poffertjie** [shinyorg/shiny#1497] [Q&A]
Hi,

I am using your Shiny.Extensions.Push library in my API to send push notifications to Firebase and  Apple. Reading the documentation on Firebase the legacy FCM APIs will be deprecated and shut down in July. Seeing as the package still needs a Serverkey for Firebase which will no longer work, is this something that will be changed in the library to accommodate these changes?

**Comment by aritchie** [comment]
It will be updated at some point in the near future
